cmd/kubeadm/app/apis/kubeadm/env.go

```diff
@@ -25,7 +25,7 @@ import (
 
 // TODO(phase2) use componentconfig
 // we need some params for testing etc, let's keep these hidden for now
-func GetEnvParams() map[string]string {
+func GetEnvParams(env map[string]string) map[string]string {
```

Comment:
When you say `types.go`, is that creating another file in the same pkg (kubeadm/app/apis/kubeadm) or some global k8s types file?